### PR TITLE
Feature/implement base crud

### PR DESCRIPTION
**Pull request by orgonek**

### CODE REVIEW

File: backend/models/token_model.py [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Token(BaseModel):
     """
     Model for representing a JWT token.
     """
+
```
Comment (daroG) [inline backend/models/token_model.py]: Docstring convention:

> There's no blank line either before or after the docstring.

File: backend/models/token_model.py [hunk 2]
```diff
@@ -13,4 +14,5 @@ class TokenData(BaseModel):
     """
     Model for representing data inside JWT token
     """
+
```
Comment (daroG) [inline backend/models/token_model.py]: same

File: backend/models/user_model.py [hunk 3]
```diff
@@ -5,6 +5,7 @@ class User(BaseModel):
     """
     Model class for representing a base user
     """
+
```
Comment (daroG) [inline backend/models/user_model.py]: same

File: backend/models/user_model.py [hunk 4]
```diff
@@ -15,11 +16,13 @@ class UserCreate(User):
     """
     Model class for creating new user
     """
+
```
Comment (daroG) [inline backend/models/user_model.py]: same

File: backend/models/user_model.py [hunk 5]
```diff
@@ -15,11 +16,13 @@ class UserCreate(User):
...
     password: str
 
 
 class UserInDB(User):
     """
     Model class for representing user stored in database
     """
+
```
Comment (daroG) [inline backend/models/user_model.py]: same

File: backend/src/recipe.py [hunk 6]
```diff
@@ -34,21 +34,21 @@ def create_recipe_from_url(self, url: str) -> bool:
         return True
 
     def print_recipe(self) -> None:
```
Comment (daroG) [inline backend/src/recipe.py]: Let's get rid of that

File: backend/src/recipes_controller.py [hunk 7]
```diff
@@ -41,19 +41,21 @@ def get_all_recipes(self) -> list[Recipe]:
...
+        """Saves recipes from given url to the database"""
+        recipes = requests.get(url).json()["recipes"]
         rec = Recipe()
         for recipe in recipes:
-            rec.create_recipe_from_url('https://www.fitczarodziejka.pl/przepis/' + recipe['slug'])
-            print('Saving recipe: ' + recipe['title'])
+            rec.create_recipe_from_url(
+                "https://www.fitczarodziejka.pl/przepis/" + recipe["slug"]
+            )
+            print("Saving recipe: " + recipe["title"])
```
Comment (daroG) [inline backend/src/recipes_controller.py]: no need (or log)

File: backend/src/recipes_service.py [hunk 8]
```diff
@@ -8,57 +8,59 @@
 
 class RecipesService:
     def __init__(self) -> None:
-        """ Initializes the database connection """
-        connection_url = "mongodb+srv://cleanuser:cleanpassword@recipescluster.bapblqj.mongodb.net" \
-                         "/?retryWrites=true"
-        self.client = pymongo.MongoClient(connection_url, server_api=ServerApi('1'))
+        """Initializes the database connection"""
+        connection_url = (
```
Comment (daroG) [inline backend/src/recipes_service.py]: No sense, we've got a configured db thah we can use here

File: backend/src/scraper.py [hunk 9]
```diff
@@ -84,3 +95,6 @@ def get_tags(self) -> list[str]:
         tags_container = self.soup.find("div", {"class": "recipeDetailTags"})
         tags = tags_container.find_all("a")
         return [tag.text for tag in tags]
+
+    def _remove_recommendation_sentence(self, text: str) -> str:
+        return re.sub("\\(polecam.*\\)", '', text).strip()
```
Comment (daroG) [inline backend/src/scraper.py]:
```suggestion
    @staticmethod
    def _remove_recommendation_sentence(text: str) -> str:
        return re.sub("\\(polecam.*\\)", '', text).strip()
```

**Review by tdxa — APPROVED** [review]

**Pull request merged**